app/assets/stylesheets/application/modules/_modal.scss

```diff
@@ -17,3 +17,20 @@ body.modal-open #wrap{
   -ms-filter: blur(15px);
   filter: blur(15px);
 }
+.organization-row {
```

Comment:
Perhaps we could move this code to another scss module, named `organization_chooser`  or something like that